.github/actions/tf-setup/action.yml

```diff
@@ -16,6 +16,9 @@ inputs:
   azure-subscription-id:
     description: The Azure subscription_id for this environment.
     required: true
+  azure-object-id:
+    description: The Azure object_id for this environment.
+    required: true
   app-name:
     description: The name of the application being deployed in Terraform.
     required: true
```

Comment:
below on the "Load input variables" we will need to add this variable to the `terraform.tfvars` file